glossary.rst

```diff
@@ -0,0 +1,54 @@
+Glossary
+========
+
+.. glossary::
+
+    Application
+        Website or application tracked by PPMS.
+
+    App ID
+        PPMS :term:`application` identificator (previously `website ID`).
+
+    User
+        Visitor on tracked :term:`application`.
+
+    Analytics ID
+        ID assigned to :term:`user` by :term:`Analytics` for the duration of :term:`Analytics` session. It's stored in
+        browser cookie.
+
+    User ID
+        Permanent ID assigned to :term:`user` by :term:`application` (e.g. username).
```

Comment:
I would add "Read more how to implement the User ID: https://help.piwik.pro/tag-manager/userid/"